README.md

```diff
@@ -337,5 +326,87 @@ Contact the dev team for the following:
 
 Once you have access to AWS LZ2 platform, login to the any of the environments to get access to the following: 
 
+- Project .env file can be found in the prod parameter store under pcc/.env
 - Database passwords
 - AWS Local Config - Service Account Tokens
+
+
+## Deployments
+
+Command Reference: 
+
+Build backend:
+```bash
+make build-backend
+```
+
+Terraform:
+```bash
+make plan
+```
+
+```bash
+make apply
+```
+Tag Based Deployments:
+```bash
+make tag-<env>
+```
+
+Deployments and builds are initiated via GitHub actions.
+We have four environments, each with a corresponding GHA yaml file:
+
+### PROD
+
+- requires an approver
+- initiate the deployment by tagging a release, ie: git tag 2.0.0 && git push origin 2.0.0
+- first, deploy this tag to dev: run make tag-dev from the command line
+    - verify the build and run any tests that should run on dev
+- next, deploy this tag to test: run make tag-test from the command line
+    - ensure that QA signs off on the release
```

Comment:
"ensure QA sign-off is complete on the release" should be listed before deploying to test (QA sign-off should be done on dev prior to deployment to test if we are using dev as testing env and QA sign-off should be done on test prior to deployment on prod)